src/platform/imx8/platform.c

```diff
@@ -156,6 +156,9 @@ int platform_init(struct sof *sof)
 
 	clock_set_freq(CLK_CPU(cpu_get_id()), CLK_MAX_CPU_HZ);
 
+	/* init IRQ_STEER */
+	platform_interrupt_init();
+
 	/* init DMA */
```

Comment:
@paulstelian97 previous patch is pretty vague for the average reviewer :D. Also, please make the commits impersonal. 
So, the commit message should look like this:

Initial implementation of platform code did not initialize the platform interrupt controller.
Fix this now by calling platform_interrupt_init() .